Lesson2/src/main/java/pl/solvd/university/administration/Deanery.java

```diff
@@ -0,0 +1,80 @@
+package pl.solvd.university.administration;
+
+import pl.solvd.university.people.Applicant;
+import pl.solvd.university.people.Student;
+import pl.solvd.university.state.ReturnDocuments;
+
+import java.io.IOException;
+import java.util.*;
+import java.util.stream.IntStream;
+
+import static pl.solvd.university.Main.*;
+
+public class Deanery implements Comparator<Applicant> {
+
+    @Override
+    public int compare(Applicant o1, Applicant o2) {
+        if (o2.getSum() == o1.getSum()) {
+            return o2.getFirstLastName().compareTo(o1.getFirstLastName());
+        } else
+            return o2.getSum() - o1.getSum();
+    }
+
+    public static void sortApplicantsByGradesAndPlaces(List<Applicant> sortedList, int NumOfBudgetPlaces, int NumOfPaidPlaces, String getSpecialty) throws Exception {
```

Comment:
what i expect from function with nam sort:
`{b, a, c} => {a, b, c}`
what this function does:
```
asks for user input if he wants to sort applications,
if yes -> sorts applications by grades
-> after sorting shows information in which form of education user is accepted
```

Reply:
Is there something wrong here? You described the implementation that is here. But it's not the end. After that, an applicant who enrolled in paid education must agree to this form of education by concluding an agreement with an accountant. If he does not do this, he is removed from the lists of applicants. And also added that since he was accepted and refused, then at the end a list of those who enrolled is still displayed, but without main applicant